utils/pdml_parser.py

```diff
@@ -59,15 +59,16 @@ def endElement(self, name):
 
     # noinspection PyMethodMayBeStatic
     def get_string(self, parsed):
-
-        parsed = parsed.replace("\t", "")
-        parsed = parsed.replace("\r", "")
-        parsed = parsed.replace("\n", "")
-        parsed = parsed.replace(",", "")
-        parsed = parsed.replace("0x", "")
-        parsed = parsed.replace("\\x", "")
-
-        value = ""
+        # until this becomes more universal, utf-8 is assumed
+        parsed = six.binary_type(parsed, "utf-8")
+        parsed = parsed.replace(b"\t", b"")
```

Comment:
In py2, using `six.binary_type(parsed, "utf-8")` will cause `TypeError`. Maybe use the buildin `helpers.str_to_bytes()` instead.

Reply:
I added that helper, do you currently use any of this utility scripts? I personally don't and often forget about them

Reply:
Me too. These scripts are copied from `Sulley` and only used for special cases.